Tests/Expected/Colors/swift2-context-defaults-customname.swift

```diff
@@ -24,18 +24,27 @@ extension Color {
 // swiftlint:enable operator_usage_whitespace
 
 // swiftlint:disable identifier_name line_length type_body_length
-enum XCTColors {
+struct XCTColors {
+  let rgbaValue: UInt32
+  var color: Color { return Color.init(named: self) }
```

Comment:
I don't know why I used `.init` there when I gave you my example, `Color(named: self)` should work I guess